globus_automate_client/cli/helpers.py

```diff
@@ -9,7 +10,11 @@
 )
 
 
-def format_and_echo(result: Union[GlobusHTTPResponse, str], verbose=False) -> None:
+def format_and_echo(
+    result: Union[GlobusHTTPResponse, str],
+    dumper: Callable = lambda result, **args: json.dumps(result, indent=4, sort_keys=True),
```

Comment:
I like this a lot, it paves the way for the day we want to support JSON/YAML output on the CLI